package.json

```diff
@@ -15,7 +15,8 @@
   "license": "GPL-3.0",
```

Comment:
delete .json files from your commit - you shouldn't commit them and push here, they should stay at your computer as local changes